clients/hadoopfs/src/test/java/io/lakefs/contract/TestLakeFSFileSystemContract.java

```diff
@@ -65,19 +63,18 @@ public void testRenameDirectoryAsExistingDirectory() throws Exception {
     Path dst = path("/test/new/newdir");
     fs.mkdirs(dst);
     rename(src, dst, true, false, true);
-    assertFalse("Nested file1 exists",
+    Assert.assertFalse("Nested file1 exists",
         fs.exists(path("/test/hadoop/dir/file1")));
-    assertFalse("Nested file2 exists",
+    Assert.assertFalse("Nested file2 exists",
         fs.exists(path("/test/hadoop/dir/subdir/file2")));
-    assertTrue("Renamed nested file1 exists",
+    Assert.assertTrue("Renamed nested file1 exists",
         fs.exists(path("/test/new/newdir/file1")));
-    assertTrue("Renamed nested exists",
+    Assert.assertTrue("Renamed nested exists",
         fs.exists(path("/test/new/newdir/subdir/file2")));
   }
 
   @Override
   public void testWorkingDirectory() throws Exception {
     // TODO make this test green and remove override
   }
-
-}
+ }
```

Comment:
Nit: trailing newline seems to have been dropped :-(

Reply:
Fixed